.changeset/coinmetrics-asset-quotes.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/coinmetrics': minor
```

Comment:
Should be `@chainlink/coinmetrics-adapter`. I suggest using the `yarn changeset` workflow (instructions [here](https://github.com/smartcontractkit/external-adapters-js/blob/develop/CONTRIBUTING.md#generating-changesets)) to make the process easier.